website/docs/reference/dbt-jinja-functions/dbt_version.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: "dbt_version"
 id: "dbt_version"
-description: "The `dbt_version` variable returns the installed version of dbt."
+description: "Read this guide to understand the dbt_version Jinga function in dbt."
```

Comment:
`Jinga` should be `Jinja`

Reply:
https://deploy-preview-3261--docs-getdbt-com.netlify.app/reference/dbt-jinja-functions/dbt_version

content="Read this guide to understand the dbt_version Jinja function in dbt."

Reply:
https://deploy-preview-3261--docs-getdbt-com.netlify.app/reference/dbt-jinja-functions/dbt_version

Read this guide to understand the dbt_version Jinja function in dbt.